src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
++++
+title = "How to deploy Istio in your cluster"
+description = "Tutorial on how to deploy Istio in the differnet providers offered by Giant Swarm."
+date = "2018-09-27"
+type = "page"
+weight = 30
+tags = ["tutorial"]
++++
+
+# Preparation
```

Comment:
The first H1 headline (one `'#`) should be the title of the article. If "Preparation" is the first content section of the article, it should be a H2 (`##`) after the article headline.

Reply:
In terms of content, I would like to greet readers who don't know a lot about Istio with one or two general sentences on what it is, why you would want to look into it and why you need special installation instructions for Giant Swarm.